### PR TITLE
Clean up PluginManagerWindow

### PR DESCRIPTION
**Pull request by InfinityGhost**

### CODE REVIEW

File: OpenTabletDriver.UX/MainForm.cs [hunk 1]
```diff
@@ -54,14 +54,12 @@ private Control ConstructPlaceholderControl()
             };
         }
 
-        public async Task Refresh()
+        public void Refresh()
```
Comment (X9VoiD) [inline OpenTabletDriver.UX/MainForm.cs:57]: OutputMode combobox should be refreshed as well.

File: OpenTabletDriver.Desktop/Reflection/DesktopPluginManager.cs [hunk 2]
```diff
@@ -135,54 +119,91 @@ where IsPluginType(type)
...
+            return result;
+        }
+
+        public bool InstallPlugin(DirectoryInfo target, DirectoryInfo source)
+        {
+            source.MoveTo(target.FullName);
+            return true;
+        }
+
+        public bool UninstallPlugin(DesktopPluginContext plugin)
```
Comment (X9VoiD) [inline OpenTabletDriver.Desktop/Reflection/DesktopPluginManager.cs:165]: During testing UX crashed on this function with the RPC exception:

```
System.NullReferenceException: Object reference not set to an instance of an object.
      at OpenTabletDriver.Desktop.Reflection.DesktopPluginManager.UninstallPlugin(DesktopPluginContext plugin)
      at OpenTabletDriver.Desktop.Reflection.DesktopPluginManager.UpdatePlugin(DesktopPluginContext plugin, DirectoryInfo source)
      at OpenTabletDriver.Desktop.Reflection.DesktopPluginManager.InstallPlugin(String filePath)
      at OpenTabletDriver.Daemon.DriverDaemon.InstallPlugin(String filePath)
```

Which happened during an update procedure of a previously uninstalled plugin.

Steps to reproduce:

1. Install Plugin.
2. Uninstall Plugin.
3. Install Plugin, and then install it again to trigger update.

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.UX/MainForm.cs (hunk 1 above)
Comment (InfinityGhost) [reply OpenTabletDriver.UX/MainForm.cs:57]: Fixed

File: OpenTabletDriver.Desktop/Reflection/DesktopPluginManager.cs (hunk 2 above)
Comment (InfinityGhost) [reply OpenTabletDriver.Desktop/Reflection/DesktopPluginManager.cs:165]: Fixed

**Review by X9VoiD — APPROVED** [review]

**Pull request merged**